src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -136,20 +151,35 @@ private List<AudioFileInfo> concatenateByChannel(List<AudioFileInfo> audioFileIn
         return processedAudios;
     }
 
-    private List<AudioFileInfo> trimToPeriod(List<AudioFileInfo> audioFileInfos, OffsetDateTime start,
-                                             OffsetDateTime end) throws ExecutionException, InterruptedException {
+    private AudioFileInfo merge(List<AudioFileInfo> audioFileInfos)
+        throws ExecutionException, InterruptedException {
+
+        return audioOperationService.merge(audioFileInfos, StringUtils.EMPTY);
+    }
+
+    private AudioFileInfo trimToPeriod(AudioFileInfo audioFileInfo, OffsetDateTime start,
+                                       OffsetDateTime end) throws ExecutionException, InterruptedException {
+        return audioOperationService.trim(
+            StringUtils.EMPTY,
+            audioFileInfo,
+            start.toLocalTime().toString(),
```

Comment:
I'm not sure this conversion to LocalTime will work here, as the `Application` sets the Default Timezone to be UTC.  The darts-api.jar also sets the system property `-Duser.timezone=UTC` in the Dockerfile.

Also the `trim` startTime, endTime strings are simply the start, stop position of the media using the ffmpeg Time duration in `HH:MM:SS` format (http://ffmpeg.org/ffmpeg-utils.html#Time-duration)

Reply:
Good spot, have updated. Please can you re-review this area? If the approach looks reasonable, would it be better to move the `OutboundFileProcessorImpl.toTimeString(Duration duration)` method into `AudioOperationService` do you think?

Reply:
Discussed with @richardbull and we can leave `toTimeString` here. If other usages in the future will benefit from it being in `AudioOperationService`, we can easily move it.

Reply:
I think it looks ok what you've done in `OutboundFileProcessorImpl#trimToPeriod`
were you thinking you would change the service in the `AudioOperationService` to overload the trim method to use `Duration `params instead?

Up to you, whether you feel it's worth the hassle - not sure about moving the toTimeString thing - maybe useful as a utility method perhaps